.deploystack/test.yaml

```diff
@@ -12,6 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-gcp_project_id = "<project_id_here>"
-
-memorystore = false
+steps:
+   - name: 'gcr.io/cloudshell-images/cloudshell:latest'
+     entrypoint: bash
+     args: [ '.deploystack/clean' ]
+   - name: 'gcr.io/cloudshell-images/cloudshell:latest'
+     entrypoint: bash
+     args: [ '.deploystack/test' ]
+timeout: 4200s
+options:
+  machineType: 'E2_HIGHCPU_8'
```

Comment:
**Question:** Why did we delete the `/terraform/terraform.tfvars` file?
We've definitely need to preserve it to support [our Terraform user journey](https://github.com/GoogleCloudPlatform/microservices-demo/tree/release/v0.4.0/terraform#deploy-the-sample-application).

Reply:
I have a process that deletes it in my dev environment.  It's possible that it got ran here by accident. But it also looks like it's present on my latest branch, so ¯\(°_o)/¯

Reply:
**Issue:**
It looks like the [`/terraform/terraform.tfvars`](https://github.com/GoogleCloudPlatform/microservices-demo/blob/release/v0.4.0/terraform/terraform.tfvars) file is missing in your [`deploystack-enable` branch](https://github.com/tpryan/microservices-demo/tree/deploystack-enable/terraform).
Could you please commit it back to your [`deploystack-enable` branch](https://github.com/tpryan/microservices-demo/tree/deploystack-enable/terraform)? :)

Reply:
**Fixed:**
Oh sweet! I was able to restore the `terraform.tfvars` file myself by commiting directly to your branch. 😮 
(I added [a commit](https://github.com/GoogleCloudPlatform/microservices-demo/pull/1142/commits#:~:text=Restore%20/terraform/terraform.tfvars) to your branch).